### PR TITLE
Allow server to calculate CFN stack name for Integrations

### PR DESCRIPTION
**Pull request by 3nvi**
## Background

This PR makes sure to allow the server to provide the `stackName` instead of thee front-end calculating it. This helps us in cases where we prompt the user to Delete or Update a particular CFN stack when updating/deleting an Integration.

The corresponding CFN stackname was not known to the front-end 

## Changes

- Add `stackName` to graphql schema
- Utilize it in all places where the front-end was using a placeholder value for it

## Testing

- Deployed to dev
- Added & Updated cloud-sec integration
- Added & Updated log-analysis integration
- Counting on @kostaspap to validate

### CODE REVIEW

**Comment by kostaspap** [comment]
Another change we wanted is that KmsKey and S3Prefix are not passed to backend to the `getIntegrationTemplate` when they are not populated. Is this being addressed in this PR, I am not sure I can see the relevant changes.

File: web/src/components/wizards/LogSourceWizard/StackDeploymentPanel/StackDeployment.tsx [hunk 1]
```diff
@@ -112,7 +98,17 @@ const StackDeployment: React.FC = () => {
             Alternatively, you can download it and deploy it through the AWS CLI with the stack name{' '}
             <b>{stackName}</b>
           </Text>
-          {downloadTemplateLink}
+          <Text size="large" color="blue300" is="span">
+            <a
+              href="#"
+              title="Download Cloudformation template"
+              download={`${stackName}.yaml`}
```
Comment (kostaspap) [inline web/src/components/wizards/LogSourceWizard/StackDeploymentPanel/StackDeployment.tsx]: We tend to name them `.yml` in our codebase, I think better follow same convention.

File: web/src/components/wizards/LogSourceWizard/StackDeploymentPanel/StackDeployment.tsx [hunk 2]
```diff
@@ -124,7 +120,17 @@ const StackDeployment: React.FC = () => {
             <Text size="large" color="grey200" mr={1}>
               1.
             </Text>
-            {downloadTemplateLink}
+            <Text size="large" color="blue300" is="span">
+              <a
+                href="#"
+                title="Download Cloudformation template"
+                download={`${initialValues.initialStackName}.yaml`}
```
Comment (kostaspap) [inline web/src/components/wizards/LogSourceWizard/StackDeploymentPanel/StackDeployment.tsx]: Same here. Rename to `.yml`

**Comment by 3nvi** [comment]
> getIntegrationTemplate

You are right! Added.

**Comment by kostaspap** [comment]
We limit the integration label to 32 characters (due to restrictions in the IAM role name size). I think it's better to have such check in the UI as well.

**Comment by 3nvi** [comment]
> We limit the integration label to 32 characters (due to restrictions in the IAM role name size). I think it's better to have such check in the UI as well.

Done

**Pull request merged**